src/components/HorizontalList/horizontal-list.css

```diff
@@ -0,0 +1,20 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+@import "src/css/variables/gutters.css";
+
+.y-horizontalList {
+  padding: 0;
+  margin: 0 -$gutterWidth__xSmall 0 0;
+  list-style: none;
+  
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.y-horizontalList__align-right {
+  justify-content: flex-end;
+}
+
+.y-horizontalList--item {
+  padding: 0;
+  margin: 0 $gutterWidth__xSmall 0 0;
```

Comment:
for consistency with our vertical rhythm, if it changes we can change it in just one place. also gutters are currently specified in `rem`; say we decide gutters should remain fixed when a user changes their browser's base font size then we could change them to `px` in one place rather than throughout the codebase in every layout component